src/lib/download.ts

```diff
@@ -0,0 +1,47 @@
+import * as https from 'https'
+import * as fs from 'node:fs'
+const progress = require('smooth-progress')
```

Comment:
I would love to someday figure out how to replace this... That's a good candidate for the design system...

Reply:
for sure.